cli/cmd/initialize.go

```diff
@@ -41,39 +48,51 @@ func handleTLSOptions() {
 			},
 		}
 		_ = survey.AskOne(modePrompt, &tlsMode)
+	} else {
+		tlsMode = Import
+	}
 
-		if tlsMode == Generate {
-			// Generate mode requires a host entry
-			prompt := &survey.Input{
-				Message: "Enter a host DNS entry or IP Address for the cluster ingress",
-			}
-			_ = survey.AskOne(prompt, &initOptions.PKI.Host, survey.WithValidator(survey.Required))
-		} else {
-			// Import mode requires the public and private key paths
-			prompt := &survey.Input{
-				Message: "Enter a file path to the ingress public key",
-				Suggest: func(toComplete string) []string {
-					// Give some suggestions to users
-					files, _ := filepath.Glob(toComplete + "*")
-					return files
-				},
-			}
-			_ = survey.AskOne(prompt, &initOptions.PKI.CertPublicPath, survey.WithValidator(survey.Required))
+	// Always ask for a host entry to avoid having to guess which entry in a cert if provided
```

Comment:
This update to always ask is now failing our e2e tests because it times out waiting for an input. This also means zarf can't be fully scripted and I feel like that's a bad thing overall.

@jeff-mccoy I'm not super familiar with networking/certs/anything.. Could you help me understand why this was changed?

Reply:
This also currently make the `--host` flag really clunky if not irrelevant.

Reply:
Nope that's a bug 😱 we need to fix it

Reply:
I assume this will get fixed already as well, just documenting here in case it really is some different issue:

If I say "Generate TLS chain with ephemeral CA", then type "localhost" for the host, it is rejected as expected. If I then type `127.0.0.1` as the host, it tells me "Enter a file path to the ingress public key" as if I had chosen to import the cert rather than generate it.

![image](https://user-images.githubusercontent.com/16000938/145631539-3fd8a371-db53-45fa-ae7c-c4262e229ba6.png)

Reply:
Similar but that was a slightly different issue. That should be fixed now.